index.js

```diff
@@ -12,11 +12,13 @@ function toBuffer (nativeStr) {
 }
 
 function init () {
-  if (RNRandomBytes.seed) {
-    let seedBuffer = toBuffer(RNRandomBytes.seed)
-    addEntropy(seedBuffer)
-  } else {
-    seedSJCL()
+  if (RNRandomBytes) {
+    if (RNRandomBytes.seed) {
+      let seedBuffer = toBuffer(RNRandomBytes.seed)
+      addEntropy(seedBuffer)
+    } else {
+      seedSJCL()
+    }
   }
```

Comment:
Shouldn't it be more like this?
```suggestion
  if (RNRandomBytes && RNRandomBytes.seed) {
    let seedBuffer = toBuffer(RNRandomBytes.seed)
    addEntropy(seedBuffer)
  } else {
    seedSJCL()
  }
```

Reply:
I am quite sure that `seedSJCL()` will continue to have the  lssues #40 , so it is necessary to use the way I used it.

Reply:
You're probably right.. I haven't spent time through the library code, I just landed on this PR as a fix to my current blocker.